majilove/magicollections.py

```diff
@@ -25,4 +27,22 @@ class DonateCollection(_DonateCollection):
     enabled = True
 
 ############################################################
+# Idol Collection
+
+class IdolCollection(MagiCollection):
+    queryset = models.Idol.objects.all()
+    title = _('Idol')
+    plural_title = _('Idols')
+    # these fields are all for after views are added
+    #navbar_title = _('Characters')
+    # want the following as 'microphone' https://www.flaticon.com/free-icon/karaoke-microphone-icon_69364
+    #icon = 'microphone'
+    #navbar_link_list = 'utapri'
```

Comment:
If you have the code ready in `majilove/settings.py` to add that navbar list, feel free to add it to this commit since it's for the collection itself.

Reply:
I think I'll put it in the navbar for now, and make it a navbar list once I add the song collection.